src/Octoshift/Services/GithubApi.cs

```diff
@@ -1003,6 +1022,36 @@ private static object GetMannequinsPayload(string orgId)
         };
     }
 
+    private static object GetMannequinsByLoginPayload(string orgId, string login)
```

Comment:
Same nitpick as above, should this be `GetMannequinByLoginPayload`?

Reply:
I think the plural is right - see my [other comment](https://github.com/github/gh-gei/pull/1067#discussion_r1268309196).